packages/design-system/src/components/icon-button-with-menu.tsx

```diff
@@ -0,0 +1,140 @@
+import React from "react";
```

Comment:
a lot of arbitrary style values instead of using the design system in this file

Reply:
Also, does this component belong into design system or into the style panel specifically?